manifest_test.go

```diff
@@ -233,15 +233,20 @@ func TestValidateManifest(t *testing.T) {
 			[[constraint]]
 			  name = "github.com/foo/bar"
 			`,
-			wantWarn:  []error{errors.New("metadata should be a TOML table")},
+			wantWarn: []error{
+				errors.New("metadata should be a TOML table"),
```

Comment:
Let's improve this. This error can be put in a variable and reused, like the variables in https://github.com/golang/dep/blob/v0.3.2/manifest.go#L31.